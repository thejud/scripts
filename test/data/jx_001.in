{"a":1, "b":"x", "c":3} 
{"a":2, "b":"z", "c":"loonoong"}
