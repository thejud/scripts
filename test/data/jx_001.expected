c         a
3         1
loonoong  2
